snscrape/modules/twitter.py

```diff
@@ -11,6 +11,7 @@
 	'TwitterTweetScraper',
 	'TwitterListPostsScraper',
 	'TwitterTrendsScraper',
+	'TwitterCashtagScraper',
```

Comment:
This should be in the right position, matching where it's defined in the file (i.e. above `TwitterTweetScraperMode`).